pygradle-plugin/src/test/groovy/com/linkedin/gradle/python/extension/PythonDefaultsTest.groovy

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.gradle.python.extension
+
+import org.gradle.api.GradleException
+import spock.lang.Specification
+import spock.lang.Unroll
+
+
+class PythonDefaultsTest extends Specification {
```

Comment:
That's the very nature of Spock!  (Both the character and package :) )

I wish Python had something as nice.  There are a few packages that try to get close, but it's difficult to do DSLs in Python.  OTOH, it's gotten me thinking about transliterating import hooks... :)